.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,26 @@
+name: Docker image build
+
+on: [push, release]
+
+jobs:
+  docker-build-push:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+      - name: Login to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Build and push
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          platforms: linux/amd64
+          push: true
+          tags: shrirajhegde/feellgood:latest
```

Comment:
If we accept this pull request, the image should be in a namespace belonging to the feeLLGood organization. I have not created an account for feeLLGood on Docker, and I don't even know whether organization accounts are even possible. Maybe it would be simpler to use the GitHub Container Registry, or GitHub releases, as this would not need creating extra accounts.

Reply:
Yes, I have enabled to accept changes from maintainers. You can push to master branch of my repository before merging, or I will make the required changes.

Docker hub supports organizations: https://docs.docker.com/admin/organization/orgs/. But I am unsure if it is available in free plan.

I just created this repository to verify successful build and push and for demonstration.